src/app/_store/auth/auth.reducer.ts

```diff
@@ -4,17 +4,20 @@ import {Action, createReducer, on} from '@ngrx/store';
 
 export interface AuthState {
   loggedIn: boolean;
+  autoLoginFailed: boolean;
   user: Partial<IIdentity> | null;
 }
 
 const initialState: AuthState = {
   loggedIn: false,
+  autoLoginFailed: false,
   user: null,
 };
 
 const authReducer = createReducer(
   initialState,
-  on(AuthActions.loginSuccess, (state, {identity}) => ({...state, loggedIn: true, user: identity})),
+  on(AuthActions.oidcAutoLoginFailure, (state, {}) => ({...state, autoLoginFailed: true})),
```

Comment:
I guess you can use `_` if you don't use this parameter but need it for typing.
```suggestion
  on(AuthActions.oidcAutoLoginFailure, (state, {}) => ({...state, autoLoginFailed: true})),
```

Reply:
Indeed, a static variable was a better solution and was enough to avoid infinite loops.